packages/react-ui/storybook/stories/molecules/Table.stories.js

```diff
@@ -0,0 +1,184 @@
+          {rows.map((row, index) => (
+            <TableRow hover key={index}>
+              <TableCell component='th' scope='row'>
+                {index + 1}
+              </TableCell>
+              <TableCell sx={{ maxWidth: 160 }}>
+                {index === 1 ? (
+                  <Tooltip title={row.name}>
+                    <Typography variant='inherit' noWrap>
+                      {row.name}
+                    </Typography>
+                  </Tooltip>
+                ) : (
+                  row.name
+                )}
+              </TableCell>
+              <TableCell>
+                <Chip size='small' color='default' label={row.type} />
+              </TableCell>
+              <TableCell>
+                <SelectField
+                  variant='filled'
+                  placeholder='Placeholder'
+                  items={[
+                    { value: '1', label: 'One' },
+                    { value: '2', label: 'Two' },
+                    { value: '3', label: 'Three' }
+                  ]}
+                />
+              </TableCell>
+              <TableCell sx={{ maxWidth: 160 }}>
+                {index === 3 ? (
+                  <Tooltip title={row.description}>
+                    <Typography variant='inherit' noWrap>
+                      {row.description}
+                    </Typography>
+                  </Tooltip>
+                ) : (
+                  row.description
+                )}
+              </TableCell>
+            </TableRow>
+          ))}
+        </TableBody>
+      </Table>
+    </TableContainer>
+  );
+};
+
+const ScrollTemplate = (args) => (
+  <Box sx={{ width: '100%', overflow: 'hidden' }}>
+    <TableContainer sx={{ maxWidth: 440, maxHeight: 300 }}>
+      <Table sx={{ minWidth: 650 }} aria-label='simple table'>
+        <TableHead>
+          <TableRow>
+            <TableCell>#</TableCell>
+            <TableCell>Name</TableCell>
+            <TableCell>Type</TableCell>
+            <TableCell>Mode</TableCell>
+            <TableCell>Description</TableCell>
+          </TableRow>
+        </TableHead>
+        <TableBody>
+          {[...rows, ...rows, ...rows, ...rows].map((row, index) => (
+            <TableRow hover key={index}>
+              <TableCell component='th' scope='row'>
+                {index + 1}
+              </TableCell>
+              <TableCell sx={{ maxWidth: 160 }}>
+                {index % rows.length === 1 ? (
+                  <Tooltip title={row.name}>
+                    <Typography variant='inherit' noWrap>
+                      {row.name}
+                    </Typography>
+                  </Tooltip>
+                ) : (
+                  row.name
+                )}
+              </TableCell>
+              <TableCell>
+                <Chip size='small' color='default' label={row.type} />
+              </TableCell>
+              <TableCell>
+                <SelectField
+                  variant='filled'
```

Comment:
Suggestion... To use the same variant and size we had before, as is the one designers are using inside a table
![Screenshot 2023-04-25 at 13 58 33](https://user-images.githubusercontent.com/1934144/234269508-454c45b4-c104-4698-a5b7-40e46de9173f.png)

So:

```
                <SelectField
                  size='small'
                  placeholder='Placeholder'
                  items={[
                    { value: '1', label: 'One' },
                    { value: '2', label: 'Two' },
                    { value: '3', label: 'Three' }
                  ]}
                />
```